Sources/ViewModels/ALKConversationViewModel.swift

```diff
@@ -1220,8 +1220,10 @@ open class ALKConversationViewModel: NSObject, Localizable {
             self.alMessages = messages.reversed() as! [ALMessage]
             self.alMessageWrapper.addObject(toMessageArray: messages)
             self.modelsToBeAddedAfterDelay = self.alMessages.map { $0.messageModel }
-
-            if self.isConversationAssignedToBot() && (UserDefaults.standard.integer(forKey: "botDelayInterval")) > 0  {
+            //Check for Conversation Assignee to show Typing Indicator
+            if !self.isConversationAssignedToBot() {
+                self.messageModels = self.modelsToBeAddedAfterDelay
+            } else if (UserDefaults.standard.integer(forKey: "botDelayInterval")) > 0 {
```

Comment:
What if the key is not present? We should check for key first, then its value

Reply:
@mukeshthawani if the key is not present then `UserDefaults.standard.integer` return 0. 
[https://developer.apple.com/documentation/foundation/userdefaults/1407405-integer](https://developer.apple.com/documentation/foundation/userdefaults/1407405-integer)

Reply:
Okay